api-spec.md

```diff
@@ -285,6 +286,42 @@ The decision route will allow partners to set a property decision for a specific
 </BackgroundReports>
 ```
 
+
+### POST : /api/partner/decision/group
+The decision route will allow partners to set a property decision for a specific group of reports. Allowed property decisions are Decline, Conditional-Low, Conditional-High, and Accept. The property decision and supplied property decision description will be used when STP generates an adverse action letter on behalf of the API partner for all group members identified in the request. Only currently grouped/linked reports can have their property decision set via this route. for an individual report the /decision route should be called.
```

Comment:
I would clarify that the verbiage is only displayed on the AAL if it's a conditional decision